Workbooks/Azure Active Directory Hybrid Auth Staged Rollout/Hybrid Auth Staged Rollout/settings.json

```diff
@@ -2,5 +2,6 @@
     "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
     "name":" Groups, Users and Sign-ins in Staged Rollout",
     "author": "Microsoft",
-    "galleries": [{ "type": "workbook", "resourceType": "microsoft.aadiam/tenant", "IsPreview":true, "order": 100 }]
-}
\ No newline at end of file
+    "IsPreview":true,

```

Comment:
`"isPreview": true,` lower case `i`, case is sensitive in these properties, as far as i know.